### DISCUSSION TITLE
How to count the number of rows in the meta data corresponding to a certain column value?

### DISCUSSION THREAD

**Discussion by pcitrano3** [satijalab/seurat#5660] [Q&A] [answered]
I am brand new to R and an analyzing single-cell RNA sequencing data. In my meta data I have a column called "RBFOX3_status". The two possible values for this column are: RBFOX3_POS or RBFOX3_NEG. Is there a way I can find out how many rows or cells contain each of those conditions? 

Essentially, I am using this to be able to subset out the RBFOX3_POS cells for further analysis, but I want to know how many of these cells there are first. 

Thank you for any help!

**Comment by samuel-marsh** [comment, marked as answer]
Hi,

You can use the following code (with object name, column name changed to your values) to return a data.frame with counts of frequency of all variables within specific column without needing to specify the specific value.  Helps when lots of potential options in column.

```r
row_count <- as.data.frame(x = table(OBJECT$COLUMN_NAME))

# Optional.  Rename column containing variable with informative name.
colnames(row_count) <- c("COLUMN_NAME", "Freq")
```

Best,
Sam
